src/host/srvinit.cpp

```diff
@@ -76,6 +76,15 @@ try
         }
     }
 
+    if (args->InConptyMode())
+    {
+        // If we're in ConPTY mode... set the no-op accessibility notifier
+        // to prevent sending expensive legacy MSAA events when we won't even be
+        // responsible for the terminal user interface.

```

Comment:
We may want to audit the places where IAccessibilityNotifier is used, in addition to having the no-op interface.

I say this because ... the one that notifies about buffer changes reads the buffer first, before calling out to IAN. It's a waste of time if IAN is a no-op, which still increases our output time burden.

Reply:
meh, you've got a good point about `SCREEN_INFORMATION::NotifyAccessibilityEventing`. If we're really trying to be as optimal as possible, then yea we shouldn't do that.

Reply:
I'm wondering why `SCREEN_INFORMATION::NotifyAccessibilityEventing` is the only thing showed up in the WPR trace. Maybe this indicates that other places that calls `IAccessibilityNotifier` are not as expensive?